packages/zoe/test/unitTests/contracts/test-priceAggregator.js

```diff
@@ -11,7 +11,6 @@ import { Far } from '@endo/marshal';
 import { makeIssuerKit, AmountMath } from '@agoric/ertp';
```

Comment:
What are the changes to this file about? They seem completely unrelated to the rest of this PR.